src/main/java/org/killbill/billing/overdue/api/OverdueCondition.java

```diff
@@ -35,4 +35,6 @@
     public PaymentResponse [] getResponseForLastFailedPaymentIn();
 
     public ControlTagType getControlTagType();
```

Comment:
@matias-aguero-hs Ok so in that case, let's make that renaming change `getControlTagType` -> `getInclusionControlTagType`, and then we should be good to merge.